front-end/src/ui-components/OnchainCreationLabel.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import styled from '@xstyled/styled-components';
+
+import Address from '../ui-components/Address';
+import TopicTag from '../ui-components/TopicTag';
+
+interface Props {
+	address?: string
+	className?: string
+	topic: string
+}
+
+const OnchainCreationLabel = ({ address, className, topic }:Props ) => {
+	return (
+		<div className={className}>
+			by
+			<Address
+				address={address || ''}
+				className='address'
+				displayInline={true}
+			/>
+				from <TopicTag className='topic' topic={topic} />
```

Comment:
where is this class `topic` used? If nowhere then we should remove it, if somewhere, then I guess it should rather be brought in this component.